### PR TITLE
Enclosures: solution

### PR DESCRIPTION
**Pull request by PastushenkoVik**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  width: 350px;
+  margin: 20px 20px 40px;
+  background-color: #308dc0;
+}
+
+.enclosures-wrap {
+  background-color: #ffa500;
+}
+
+.enclosures-p {
```
Comment (YuriiHoliuk) [inline src/style.css]: not the best class name

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.enclosures-border {
```
Comment (YuriiHoliuk) [inline src/style.css]: class - is name of the element - this is not a border, this is a block or box etc.

**Review by YuriiHoliuk — COMMENTED** [review]
All is good, but think about naming

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 10px;
+}
+
+.enclosures {
+  width: 350px;
+  margin: 20px 20px 40px;
+  background-color: #308dc0;
+}
+
+.enclosures-wrap {
```
Comment (irenhh) [inline src/style.css]: use `enclosures-wrapper` instead of `...-wrap` (better to use nouns for names, not verbs)

**Review by irenhh — APPROVED** [review]
The comments below just for considering

**Pull request closed without merging**